### PR TITLE
Add fields validation when creating or updating the project

### PR DESCRIPTION
**Pull request by chacha912**
**What this PR does / why we need it**:

- [x] Add project name validation when creating a project
- [x] Add AuthWebhookURL validation when updating a project 
- [x] Print error details when creating/updating a project with command

AS-IS
- don't know which fields are violated.
```bash
$ bin/yorkie project update news --name=new

# Error: rpc error: code = InvalidArgument desc = invalid fields
```

TO-BE
```bash
$ bin/yorkie project update news --name=new

# Invalid Fields: The "Name" field was wrong: given Name is reserved name
# Error: rpc error: code = InvalidArgument desc = invalid fields
```
image

**Which issue(s) this PR fixes**:

Address #354 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation**:

```docs

```

**Checklist**:
- [x] Added relevant tests or not required
- [x] Didn't break anything

### CODE REVIEW

File: admin/client.go [hunk 1]
```diff
@@ -169,6 +172,16 @@ func (c *Client) CreateProject(ctx context.Context, name string) (*types.Project
...
+		st := status.Convert(err)
+		for _, detail := range st.Details() {
+			switch t := detail.(type) {
+			case *errdetails.BadRequest:
+				for _, violation := range t.GetFieldViolations() {
+					fmt.Printf("Invalid Fields: The %q field was wrong: %s\n", violation.GetField(), violation.GetDescription())
+				}
+			}
+		}
+
```
Comment (hackerwins) [inline admin/client.go:184]: So far, I/O has been handled by the CLI(packages in the `cmd`).

https://github.com/yorkie-team/yorkie/blob/f11a6ed1c207f55fed92f0cc0d16d027104afbf7/cmd/yorkie/document/list.go#L74-L84

For consistency, I think it's better to return an error here and handle the actual output from the CLI. 🤔

**Review by hackerwins — COMMENTED** [review]
Overall looks good. 👍 
I left a comment.

File: admin/client.go (hunk 1 above)
Comment (chacha912) [reply admin/client.go]: Thanks. I changed to handle in the cmd.

File: cmd/yorkie/project/create.go [hunk 2]
```diff
@@ -54,6 +55,15 @@ func newCreateCommand() *cobra.Command {
 			ctx := context.Background()
 			project, err := cli.CreateProject(ctx, name)
 			if err != nil {
+				st := status.Convert(err)
```
Comment (hackerwins) [inline cmd/yorkie/project/create.go:58]:
```suggestion
				// TODO(chacha912): consider creating the error details type to remove the dependency on gRPC.
				st := status.Convert(err)
```

File: cmd/yorkie/project/update.go [hunk 3]
```diff
@@ -83,6 +84,16 @@ func newUpdateCommand() *cobra.Command {
 
 			updated, err := cli.UpdateProject(ctx, id, updatableProjectFields)
 			if err != nil {
+				st := status.Convert(err)
```
Comment (hackerwins) [inline cmd/yorkie/project/update.go:87]:
```suggestion
				// TODO(chacha912): consider creating the error details type to remove the dependency on gRPC.
				st := status.Convert(err)
```

**Review by hackerwins — APPROVED** [review]
LGTM. 👍

**Pull request merged**